src/conic_algorithm.jl

```diff
@@ -117,7 +118,7 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
     obj_best::Float64           # Best conic (feasible) objective value
     gap_rel_opt::Float64        # Relative optimality gap = |obj_mip - obj_best|/|obj_best|
     soln_best::Vector{Float64}  # Best original solution vector (corresponding to best objective)
-    queue_heur::Vector{Vector{Float64}} # Heuristic queue for x_all
+    queue_heur::Vector{Vector{Vector{Float64}}} # Heuristic queue for MIP solutions (if mip_solver_drives)
```

Comment:
because I save [int var vals, cont var vals, current b conic vector]. 

could do queue_heur_int, queue_heur_cont, queue_heur_b instead, but more wordy